include/contractor/graph_contractor.hpp

```diff
@@ -336,15 +336,15 @@ class GraphContractor
     void Run(double core_factor = 1.0)
     {
         // for the preperation we can use a big grain size, which is much faster (probably cache)
-        const constexpr size_t InitGrainSize = 100000;
-        const constexpr size_t PQGrainSize = 100000;
+        constexpr size_t InitGrainSize = 100000;
+        constexpr size_t PQGrainSize = 100000;
         // auto_partitioner will automatically increase the blocksize if we have
         // a lot of data. It is *important* for the last loop iterations
         // (which have a very small dataset) that it is devisible.
-        const constexpr size_t IndependentGrainSize = 1;
-        const constexpr size_t ContractGrainSize = 1;
-        const constexpr size_t NeighboursGrainSize = 1;
-        const constexpr size_t DeleteGrainSize = 1;
+        constexpr size_t IndependentGrainSize = 1;
+        constexpr size_t ContractGrainSize = 1;
+        constexpr size_t NeighboursGrainSize = 1;
+        constexpr size_t DeleteGrainSize = 1;
```

Comment:
Hmm ... I'm still a little confused.
What I found on the subject is this:
https://isocpp.org/wiki/faq/cpp14-language#extended-constexpr

Where it says:
C++14 also removes the C++11 rule that constexpr member functions are implicitly const.

But that does not apply to local variables.